README.md

```diff
@@ -303,6 +303,7 @@ Here is a list of the most essential security and privacy enhancing add-ons that
 * [DuckDuckGo Plus](https://addons.mozilla.org/firefox/addon/duckduckgo-for-firefox/) (instead of Google)
 * [No Resource URI Leak](https://addons.mozilla.org/firefox/addon/no-resource-uri-leak/) (see [#163](https://github.com/pyllyukko/user.js/issues/163))
 * [Decentraleyes](https://addons.mozilla.org/firefox/addon/decentraleyes/)
+* [Canvas Blocker](https://addons.mozilla.org/firefox/addon/canvasblocker/) ([Source code](https://github.com/kkapsner/CanvasBlocker))
```

Comment:
I have been using for months; it appears to work as advertised on the addon page, and is currently (I  think?) the only defense against canvas fingerprinting (sending fake readouts instead of completely disabling canvas).

I have set it to always fake readouts, and be silent with `extensions.CanvasBlocker@kkapsner.de.showNotifications: false` (otherwise it shows an infobar asking what to do for each page that uses canvas)